docs: documenting using pymapdl on clusters (#3466)
ansys/pymapdl: doc/source/user_guide/mapdl.rst

=====================

There are several PyMAPDL-specific environment variables that can be
used to control the default behavior of PyMAPDL or launching MAPDL.

> It should be mentioned that these environment variables do not have
> priority over the arguments given in the corresponding functions.
> For instance:

Reviewer: ```suggestion
These environment variables do not have
priority over the arguments given in the corresponding functions:
```

.. code-block:: console
    
    user@machine:~$ export PYMAPDL_PORT=50052
    user@machine:~$ python -c "from ansys.mapdl.core import launch_mapdl; mapdl=launch_mapdl(port=60053)"

> The preceding command launches an MAPDL instance on the port 60053,
> because the argument ``port`` has priority over the environment
> variable :envvar:`PYMAPDL_PORT`.
> 
> These are described in the following table:

Reviewer: ```suggestion
This command launches an MAPDL instance on port 60053
because the ``port`` argument has priority over the :envvar:`PYMAPDL_PORT`
environment variable. The following table describes all arguments.

```
